src/analysis/convex/exposed.lean

```diff
@@ -0,0 +1,201 @@
+/-
+Copyright (c) 2021 Yaël Dillies, Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies, Bhavik Mehta
+-/
+import analysis.convex.extreme
+
+/-!
+# Exposed sets
+
+This file defines exposed sets and exposed points for sets in a real vector space.
```

Comment:
It would be nice to explain what you mean by exposed sets here: see eg the docstrings for convex/basic and convex/cone